README.md

```diff
@@ -29,6 +29,28 @@ npm run serve
 TV_IP=10.0.1.1 npm run test-tv
 ```
 
+## Adding Throttling
+
+If the test environment is too good, and the metrics are too big to catch dips in performance after a PR you can test it with `CPU throttling` of 2 with the commands:
+
+```
+npm run test-throttling
+```
+
+```
+npm run test-tv-throttling
+```
+
+or for a custom value use the commands:
+
+```
+npm run test --throttling=4

```

Comment:
Isn't it `npm run test -- --throttling=4`?

Reply:
I changed the command to use the structure you suggested